### PR TITLE
Api600 enclosures

### PR DESCRIPTION
**Pull request by madhav-bharadwaj**
### Description
Support for api600 enclosures

### Check List
- [X] New functionality includes testing.
  - [X] All tests pass (`$ rake test`).
- [X] New functionality has been documented in the README if applicable.
  - [X] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [X] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by jyothisgm — APPROVED** [review]
LGTM

File: lib/puppet/provider/oneview_enclosure/c7000.rb [hunk 1]
```diff
@@ -102,4 +102,28 @@ def patch_enclosure
...
+    true
+  end
+
+  def import_csr
+    bay_number = @data.delete('bay_number')
+    enclosure = get_single_resource_instance
+    @options = enclosure.get_csr_request(bay_number)
+    @data[:type] = @options.delete('type')
+    @data[:base64Data] = @options.delete('base64Data')
+    pretty @data
```
Comment (soodpr) [inline lib/puppet/provider/oneview_enclosure/c7000.rb]: is this required?

Comment (madhav-bharadwaj) [reply lib/puppet/provider/oneview_enclosure/c7000.rb]: No not required. My bad. I'll remove it. Was using it while developing. Thanks.

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**